code/game/gamemodes/objective.dm

```diff
@@ -240,9 +240,8 @@
 		return 0
 	if(issilicon(owner.current))
 		return 0
-	if(SSshuttle.emergency.shuttle_areas[get_area(owner.current)])
+	if(!(SSshuttle.emergency.shuttle_areas[get_area(owner.current)]))
```

Comment:
I assumed I was supposed to? I think I saw it somewhere else in the code and figured it was some standard :S.